esbuild/copy.js

```diff
@@ -7,10 +7,12 @@ module.exports = async function( baseConfig ) {
   const resolvedBase = resolve( unprocessed );
   const files = await getFiles( resolvedBase );
 
-  const staticFiles = files.filter( v => !v.match( /^\.|\.js$|\.less$|\.css$/ ) );
+  const staticFiles = files.filter( v => !v.match( /\/\.[-.\w]*$|\.js$|\.less$|\.css$/i ) );
```

Comment:
For future maintenance, maybe add a comment above this line with a link to a regex example showing what this is matching and not matching, like is done here https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/js/modules/util/validators.js#L35